sky/server/server.py

```diff
@@ -478,13 +479,18 @@ async def launch(launch_body: payloads.LaunchBody,
 # pylint: disable=redefined-builtin
 async def exec(request: fastapi.Request, exec_body: payloads.ExecBody) -> None:
```

Comment:
The question is here is whether we should support an argument like `--no-wait` to bypass precondition (which is same as the previous behavior).

^ I think not because it can be confusing with `--async` and I do not think of a scenario where I I would need such arguments.